components/CarrouselAlliances.tsx

```diff
@@ -34,10 +33,15 @@ export default function CarrouselAlliances ({ allies = [], hasContent }: Props)
   return (
     <div className='carrousel'>
       <div className={classnames(
-        'carrousel-alliances',
-        {
-          'no-content': !hasContent
-        }
+        'hidden lg:block relative',
+        'mt-12 mb-5',
+        'before:left-0 before:top-0',
+        'after:right-0 after:top-0 after:transform after:rotate-180',
+        'before:absolute before:w-24 before:h-32 before:content-[""] before:z-20',
+        'after:absolute after:w-24 after:h-32 after:content-[""] after:z-20',
+        'before:bg-gradient-to-r from-brand-black to-transparent',
+        'after:bg-gradient-to-r from-brand-black to-transparent',
+        `${!hasContent && 'before:content-none after:content-none'}`,
```

Comment:
A classnames le puedes pasar un objeto en cualquiera de sus parametros, si el valor de la key es verdadero, renderiza la key como una clase, si el valor es falsy no se renderiza nada, entonces debería quedar algo así:

```suggestion
        { 'before:content-none after:content-none': !hasContent }
```

Así como lo vemos en [su documentación](https://www.npmjs.com/package/classnames)